pkg/util/math.go

```diff
@@ -19,3 +20,12 @@ func Max(x, y int64) int64 {
 func MaxDuration(x, y time.Duration) time.Duration {
 	return time.Duration(Max(int64(x), int64(y)))
 }
+
+func GenerateAlphaNumeric(len int) string {
+	bytes := make([]byte, len)
+	for i := 0; i < len; i++ {
+		bytes[i] = byte(65 + rand.Intn(25))
+
+	}
+	return string(bytes)
```

Comment:
This might fail due to the fact that you are not ensuring proper UTF-8 with the random bytes. But I would recommend anyway to drop this in favor of `util.GeneratePassword()`.

Reply:
Done